### PR TITLE
[MRG + 1] DOC

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Slightly more didactic docs

### CODE REVIEW

File: doc/decoding/decoding_intro.rst [hunk 1]
```diff
@@ -64,36 +73,61 @@ corresponding category.
...
+
+   The Haxby experiment is unusual because the experimental paradigm is
+   made of many blocks of continuous stimulation. Most cognitive
+   experiments have a more complex temporal structure with rich sequences
+   of events.
 
-Loading the data into Python
-----------------------------
+   The standard approach to decoding consists in fitting a first-level
+   GLM to retrieve one responce map (a beta map) per trial. This is
```
Comment (bthirion) [inline doc/decoding/decoding_intro.rst]: response

File: doc/decoding/decoding_intro.rst [hunk 2]
```diff
@@ -64,36 +73,61 @@ corresponding category.
...
-* **Fetching the data**: In the tutorial, we load the data using nilearn
-  data downloading function, :func:`nilearn.datasets.fetch_haxby` (if the
-  data is not present on the disk, it can take a while to download about
-  310 Mo of data from the Internet):
-
-  It return an object with several entries that contain paths to the
-  files downloaded on the disk.
+* **Retrieving the data**: In the tutorial, we load the data using nilearn
+  data downloading function, :func:`nilearn.datasets.fetch_haxby`.
+  However, all this function does is downloading the data and return
```
Comment (bthirion) [inline doc/decoding/decoding_intro.rst]: 'downloading and returning' or 'download and return'

File: doc/decoding/decoding_intro.rst [hunk 3]
```diff
@@ -64,36 +73,61 @@ corresponding category.
...
+* **Extracting the fMRI data**: we then use the
+  :class:`nilearn.input_data.NiftiMasker`: we extract only the voxels on
+  the mask of the ventral temporal cortex that comes with the data,
+  applying the `mask_vt` mask to
+  the 4D fMRI data. The resulting data is then a matrix with a shape that is
+  (n_timepoints, n_voxels)
   (see :ref:`mask_4d_2_3d` for a discussion on using masks).
 
-* **Sample mask**: Masking some of the data points may be useful to
+* **Sample mask**: Masking some of the times points may be useful to
```
Comment (bthirion) [inline doc/decoding/decoding_intro.rst]: time

**Comment by bthirion** [comment]
LGTM.

**Comment by GaelVaroquaux** [comment]
@bthirion : I've addressed your comments.

I've finished making the changes to the documentation that came to my head during the Stockholm tutorial. This is ready to merge.

File: doc/decoding/decoding_intro.rst [hunk 4]
```diff
@@ -247,7 +286,8 @@ to use the session label, present in the behavioral data file, and
     >>> classification_accuracy  # doctest: +SKIP
     0.76851...
 
-We have a total prediction accuracy of *77%* across the different sessions.
+For disriminating human faces from cats, we measure a total prediction
```
Comment (KamalakerDadi) [inline doc/decoding/decoding_intro.rst]: disriminating -> discr

File: doc/decoding/decoding_intro.rst [hunk 5]
```diff
@@ -257,9 +297,10 @@ the number of total errors. It is not always a sensible metric,
 especially in the case of very imbalanced classes, as in such situations
 choosing the dominant class can achieve a low number of errors.
 
-Other metrics, such as the f1-score, can be used::
+Other metrics, such as the AUC (Aera Under the Curve, for the ROC: the
```
Comment (KamalakerDadi) [inline doc/decoding/decoding_intro.rst]: Aera -> Area

File: doc/decoding/decoding_intro.rst [hunk 6]
```diff
@@ -257,9 +297,10 @@ the number of total errors. It is not always a sensible metric,
...
+Reciever Operating Characteristic), can be used::
```
Comment (KamalakerDadi) [inline doc/decoding/decoding_intro.rst]: Reciever -> Reciever

File: doc/decoding/estimator_choice.rst [hunk 7]
```diff
@@ -209,7 +209,22 @@ little guarantee on the brain maps.
...
+
 .. seealso::
 
    * :ref:`space_net`
 
+|
+
+.. topic:: **Decoding on simulated data**
+
+   Simple simulations may be useful to understood the behavior of a given
```
Comment (KamalakerDadi) [inline doc/decoding/estimator_choice.rst]: understood -> understand

File: doc/decoding/estimator_choice.rst [hunk 8]
```diff
@@ -209,7 +209,22 @@ little guarantee on the brain maps.
...
+   decoder on data. In particular, simulations enable us to set the true
+   weight maps and compare them to the ones retrieved by decoders. A full
+   example running simulations and discussing them can be found in
+   :ref:`sphx_glr_auto_examples_02_decoding_plot_simulated_data.py`
+   Simulated data cannot easily mimic all properties of brain data. An
+   important aspect, however, is its spatial structure, that we create in
+   the similations.
```
Comment (KamalakerDadi) [inline doc/decoding/estimator_choice.rst]: similations -> simulations

File: examples/02_decoding/plot_haxby_anova_svm.py [hunk 9]
```diff
@@ -21,32 +21,34 @@
...
-y = labels['labels']
-session = labels['chunks']
+# Load target information as string and give a numerical identifier to each
+behavioral = np.recfromcsv(haxby_dataset.session_target[0], delimiter=" ")
+conditions = behavioral['labels']
 
-# Restrict to faces and houses
-condition_mask = np.logical_or(y == b'face', y == b'house')
-y = y[condition_mask]
+# Restrict the analysis to faces and places
```
Comment (KamalakerDadi) [inline examples/02_decoding/plot_haxby_anova_svm.py]: places -> animals

File: examples/02_decoding/plot_haxby_anova_svm.py [hunk 10]
```diff
@@ -71,55 +73,52 @@
...
 coef = svc.coef_
 # reverse feature selection
 coef = feature_selection.inverse_transform(coef)
 # reverse masking
-weight_img = nifti_masker.inverse_transform(coef)
+weight_img = masker.inverse_transform(coef)
 
 
-# Create the figure
+# USe the mean image as a background to avoid relying on anatomic data
```
Comment (KamalakerDadi) [inline examples/02_decoding/plot_haxby_anova_svm.py]: USe -> Use
anatomic -> anatomical

**Comment by GaelVaroquaux** [comment]
Thanks for the comments. I addressed all of them.

**Comment by KamalakerDadi** [comment]
+1 for merge.

**Comment by GaelVaroquaux** [comment]
Thanks for the review!

**Pull request merged**